### ISSUE TITLE
Nvidia Shield SPMC 720p

### ISSUE THREAD

**Issue by bturner75** [koying/SPMC#692]
I'm not sure if this is an issue or if I'm missing a setup step (which very well may be the case.) Just got the new Nvidia Shield.  I have SPMC on my older shield connected to my 1080p TV and it works great.
The new shield I connected, for now, to my 720p TV.  however when I try to change the resolution in system settings the only option there is 1920x1080.  The movies also play like they are playing in 1080p resolution so 2.35 movies are "squished" because it is a 720p TV displaying in 1080p I believe?
Any help would be appreciated
Thanks

**Comment by sask3m** [comment]
I'm having the same issue both shield tv and a nexus player.  Shows 1920 x 1080 resolution on 1080i tv and in the drop down tab is simply another 1920 x 1080 to select where it should be something like 120 x 720.  This on spmc v 16.5.5. Thanks.

**Comment by bturner75** [comment]
I was able to change the "size" of the video when it is playing - forgot about the setting. However in the system setup for the GUI resolution I still can only have it display 1920x1080.  Even if I set the limit to 1080.  Because of this I have to do a zoom so I can see all of the menu options.

**Comment by sask3m** [comment]
By changing the size of the video are you referring to the calibration option?  I noticed in Kodi 17 I have the option of the gui in 1080p or 720p.

**Comment by bturner75** [comment]
No by size of the video I meant the view mode from the OSD when playing a video. It is under video settings. On my 720p TV when I had it set to "normal" it was, for some reason, displaying it like it was 1080p (a 1.85 movie had black bars at the bottom- which I'm fine with but on my 720p TV they usually are not there.) 
when I changed the view mode to "original size" it looks as I would expect it to when playing a video.

But yeah in my SPMC 16.5.5 in my GUI settings I only have the option to do 1920x1080.

**Comment by sask3m** [comment]
I see now how to change the GUI setting to 1280 x 720.  Under
System/settings/system/video output/ set GUI resolution.  I then went and played around with the video calibration settings.

  Too many settings. :)

**Comment by bturner75** [comment]
Thanks I may have to check that out.   I was incorrect with my original thought about changing the "view mode" to original size.  It works on some videos but not all.
They work and display fine though on other movie apps just not SPMC.

**Comment by bturner75** [comment]
Yeah that didn't work for me. I have to do a zoom in the appearance setting to get the gui to display everything correctly.
Movies still display incorrectly. Spmc thinks my tv and resolution is 1080p and it is a 720p I believe.
Plex and Kodi playback the movies correctly.
Not sure what's going o

**Comment by bturner75** [comment]
Well that is odd. I uninstalled and reinstalled and the movies now play correctly and look correct.
The GUI still won't let me choose 1280x720p though so I have to use Zoom.
